py/tests/test_processing.py

```diff
@@ -173,3 +173,43 @@ def test_parse_release():
 def test_parse_release_error():
     with pytest.raises(sentry_relay.InvalidReleaseErrorBadCharacters):
         sentry_relay.parse_release("/var/foo/foo")
+
+
+@pytest.mark.parametrize(
+    "condition",
+    [
+        '{"op": "eq", "name": "field_1", "value": ["UPPER", "lower"], "options":{ "ignoreCase": true}}',
+        '{"op": "eq", "name": "field_2", "value": ["UPPER", "lower"]}',
+        '{"op": "glob", "name": "field_3", "value": ["1.2.*", "2.*"]}',
+        '{"op": "not", "inner": {"op": "glob", "name": "field_4", "value": ["1.*"]}}',
+        '{"op": "and", "inner": [{"op": "glob", "name": "field_5", "value": ["2.*"]}]}',
+        '{"op": "or", "inner": [{"op": "glob", "name": "field_6", "value": ["3.*"]}]}',
+        '{"op": "custom", "name": "event:legacy_browser", "value": ["ie9", "ie10", "ie11"]}',
+    ],
+    ids=("eq with case", "eq", "glob", "not", "and", "or", "custom"),
```

Comment:
All of these should be unit tested. Let's only leave a single test here to test the binding.

Reply:
The operators are unit tested. 

What I test here is that the validator works as expected, not the functionality of the operator and not even the serialization of the structure (although the validator effectively uses serialization to do its work).

Reply:
To clarify: Let's please put these tests in Rust code, ideally into the module that declares the types inside `relay-sampling`. Then, only keep a single Python test here that asserts the wrapper works.

Reply:
got it